02-GitHub.Rmd

```diff
@@ -9,67 +9,74 @@ opts_chunk$set(tidy.opts=list(width.cutoff=50), tidy = FALSE)
 
 Trying to find instructions on how to use git and GitHub online is notoriously difficult for beginners. You'll often come across instructions like this:
 
-> Doing X is easy. All you have to do is open up a terminal and type git -s -x -what -thehell -doesthisallevenmean
+> Doing X is easy. All you have to do is open up a terminal and type 
+>
+> `git -s -x -what -thehell -doesthisallevenmean`
 
-Thankfully, using git and GitHub is super easy and you don't have to use the command line at all to benefit from these tools.
```

Comment:
I've always heard that it's a bad idea to say that things are 'easy' when teaching, because students feel like they're failing if they find it hard, so perhaps reverse it being 'not that hard' as opposed to 'super easy'?

Reply:
Good suggestion.